diff --git a/node_modules/wuchale/dist/adapter-utils/mixed-visitor.js b/node_modules/wuchale/dist/adapter-utils/mixed-visitor.js
index 0a7d3e5..91d962d 100644
--- a/node_modules/wuchale/dist/adapter-utils/mixed-visitor.js
+++ b/node_modules/wuchale/dist/adapter-utils/mixed-visitor.js
@@ -260,7 +260,10 @@ export class MixedVisitor {
             }
             else if (props.commentDirectives.forceType !== false) {
                 if (this.#props.leaveInPlace(child)) {
+                    this.#mod[props.scope] = newMod();
                     msgs.push(...this.#props.visitFunc(child));
+                    msgs.push(...this.applyMod(props.scope));
+                    this.#mod[props.scope] = mod;
                 }
                 else if (this.#props.isExpression(child)) {
                     msgs.push(...this.#props.visitFunc(child));
